pkg/pod/pod_test.go

```diff
@@ -65,6 +65,14 @@ func TestMakePod(t *testing.T) {
 		VolumeMounts: []corev1.VolumeMount{toolsMount},
 	}
 
+	resultsInit := corev1.Container{
```

Comment:
So… this is a *kinda **big*** problem on OpenShift as… it fails (running as non-root, …)

Reply:
Btw, I don't think this one is actually needed (at least, from the OpenShift point of view, the `emptyDir` should make the path writable)

Reply:
Sorry about the extra churn from the chmod.  You're right - emptyDir gets 777 perms: https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/emptydir/empty_dir.go#L38-L43